packages/rocketchat-language-plugin/server/functions/leave_automatic_channel.js

```diff
@@ -0,0 +1,18 @@
+RocketChat.leave_automatic_channel = function(room_name) {
+	const is_channel_automatic = Meteor.users.find({
+		'automatic_channels.name':room_name
```

Comment:
Why this way of testing if channel is automatic? This should just be a flag in the room/channel document.